### ISSUE TITLE
[UI] Should be able to download signed zip package when in closed state (after approval)

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/dac-ui#446]
When an application has been `approved`, then is `closed`, the applicant and daco admin should be able to download the signed zip package that was submitted for DACO review.

- [ ] this should say "SIGNED PDF"
- [ ] this should download the same pdf that could be downloaded on the approved application (the zipped package of what the applicant submitted)

![image](https://user-images.githubusercontent.com/18424138/134233744-e5360f86-16f7-49ae-8959-b246b68ef7e9.png)

**Comment by kcullion** [comment]
@ciaranschutte I don't know if this is working as expected. Here's what I noticed:
1. I filled out an application, uploaded a couple ethics files.
2. in sign and submit section, I downloaded the finalized pdf and renamed it with _signed at the end. Then I submitted
3. I approved the application as DACO admin
4. I logged back in as the applicant and tested the pdf download, it downloaded that pdf with the _signed at the end, as well as all of the ethics letters I had uploaded on my application. 
5. I closed the application
6. The pdf button has the correct label ("SIGNED PDF") but when I click on it it doesn't download that same package of signed pdf and ethics letters. It downloads just the finalized pdf again. 

Expected - it should download the same zip package as it does before closing. 
![image](https://user-images.githubusercontent.com/18424138/135916815-fe976b45-103d-4a46-a77b-9892d9cd936c.png)
